features/buyer/direct-award.feature

```diff
@@ -4,9 +4,5 @@ Feature: Direct Award flows
 
 Scenario: User can save a search into a new Direct Award Project
   Given I am logged in as a buyer user
-  And I am on the /g-cloud/search page
-  And I click 'Save search'
-  Then I am on the 'Save your search' page
-  And I enter 'my cloud project' in the 'Name your search' field
-  And I click 'Save and continue'
+  And I have a saved search
   Then I am on the 'my cloud project' page
```

Comment:
This test doesn't really make sense with the steps that it ends up with. It should have the steps in there properly so that it reads like a complete scenario. It might create some duplication, but it just doesn't make sense as it is: `Given I am logged in as a buyer, And I have a saved search, Then I am on the 'my cloud project' page.`

Reply:
Agree, I think if you really want to deduplicate then you might come up with a better name for the shared step, perhaps instead of `Given(/^I have a saved search$/) ` you could have `When(/^I create and save a search$/) `